{{cookiecutter.project_slug}}/compose/local/django/celery/flower/start

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o xtrace
+
+
+celery flower -A {{cookiecutter.project_slug}}.taskapp worker --broker="${CELERY_BROKER_URL}"
```

Comment:
The `worker` is not needed here?

Agree adding a basic auth for production.

Reply:
Sure, just copy-pasted existing `celeryd` commands :)